src/useAsyncEffect.js

```diff
@@ -1,6 +1,6 @@
 import { useEffect } from 'react'
 
-const useAsyncEffect = (effect, inputs, destroy) => {
+export default function (effect, inputs, destroy) {
```

Comment:
I'm not sure why you did this, but you should delete the export default useAsyncEffect on the bottom of this file, when you do this.

Reply:
You are right! I just wanted to have consistency in the repo so that every file uses or the const syntax or the function syntax. With being fast about it I was sloppy and forgot to remove the export at the bottom of the file.